packages/api-client/src/conversation/ConversationAPI.ts

```diff
@@ -791,6 +793,25 @@ export class ConversationAPI {
     const response = await this.client.sendProtocolBuffer<ClientMismatch>(config, true);
     return response.data;
   }
+  /**
+   * Post an encrypted message to a conversation.
+   * @param messageData Mls message payload in TLS format. Please refer to the MLS specification for details.
+   * @see https://messaginglayersecurity.rocks/mls-protocol/draft-ietf-mls-protocol.html#name-message-framing
+   * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/conversations/postOtrMessage
+   */
+  public async postMlsMessage(messageData: Uint8Array): Promise<MlsEvent> {
+    const config: AxiosRequestConfig = {
+      data: {
+        // TODO: use core crypto here to encrypt the message data
```

Comment:
Side note: The core-crypto lib will never be known from the api-client, it's going to live in the `cryptoRepo` of the `core` ;)

Reply:
I removed this comment